src/components/HeatmapGrid.js

```diff
@@ -36,37 +23,33 @@ const RowHeader = styled.td`
   font-size: ${({ theme }) => theme.font.size.medium};
 `;
 
-const HeatmapGrid = ({ posts }) => {
-  const postCountByHour = useMemo(() => getPostCountByHour(posts), [posts]);
-  const [clickedCellId, setClickedCellId] = useState(null);
+const HeatmapGrid = ({ postCountByHour, clickedCellId, setClickedCellId }) => (
```

Comment:
`postCountByHour` can be derived from `postsByHour`. So you could use the `postsByHour` here as well instead of `postCountByHour`. Then you could remove the [countsByHour state in useFetchPosts](https://github.com/ooloo-io/reddit-timer-estuelke/blob/5438fea86a07af355944abc2b2c9f9fa6c37bb00/src/hooks/useFetchPosts.js#L54). 

Below instead of `hourCount` you could use `posts.length`.